tests/e2e/tutorials/jupyters.js

```diff
@@ -72,7 +78,13 @@ async function runTutorial() {
 
     const iframeHandles2 = await tutorial.getIframe();
     // expected three iframes = loading + jupyterNB + jupyterLab
-    const jLabIframe = await iframeHandles2[2].contentFrame();
+    const iFrame20 = await iframeHandles2[0].contentFrame();
+    const iFrame21 = await iframeHandles2[1].contentFrame();
+
+    var jLabIframe = iFrame20;
```

Comment:
```suggestion
    var jLabIframe = iFrame20;
```
    let jLabIframe = iFrame20;